src/site/antora/modules/ROOT/pages/log4j-spring-cloud-config-client.adoc

```diff
@@ -122,67 +126,9 @@ logging:
 Note that Log4j currently does not directly support encrypting the password.
 However, Log4j does use Spring's  standard APIs to access properties in the Spring configuration so any customizations made to Spring's property handling would apply to the properties Log4j uses as well.
 
-If more extensive authentication is required an `AuthorizationProvider` can be implemented and the fully qualified class name in the `log4j2.authorizationProvider` system property, in log4j2.component.properties or in Spring's bootstrap.yml using either the `log4j2.authorizationProvider` key or with the key `logging.auth.authorizationProvider`.
+If more extensive authentication is required an `AuthorizationProvider` can be implemented and the fully qualified class name in the xref:manual/configuration.adoc#log4j2.configurationAuthorizationProvider[log4j2.configurationAuthorizationProvider] system property, in `log4j2.component.properties` or in Spring's `bootstrap.yml` using either the `log4j2.authorizationProvider` key or with the key `logging.auth.authorizationProvider`.
```

Comment:
Neither do I. ;-)

Reply:
That is because it is missing a few words. It should read "If more extensive authentication is required an `AuthorizationProvider` can be implemented and the fully qualified class name should be specified in the xref:manual/configuration.adoc#log4j2.configurationAuthorizationProvider[log4j2.configurationAuthorizationProvider] system property, in `log4j2.component.properties` or in Spring's `bootstrap.yml` using either the `log4j2.authorizationProvider` key or with the key `logging.auth.authorizationProvider`."

That said, this is one of thing I am not crazy about the design of. I would prefer that the AuthorizationProvider be injected with the DI system.

Reply:
@rgoers, thanks.

Fixed in 928d23bfdb70b3f346856fc7c12ffcf6bfb0d196